network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/generic-card.html

```diff
@@ -0,0 +1,16 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div class="child-card col-6 col-md-4 mb-5">
+  <div>
+    <a href="{{ page.url }}"">
```

Comment:
extra `"`

Reply:
Does it need quotation marks at all though?

Reply:
it does, this isn't React JSX, Django templating does not automagically add the necessary quotes because it has no idea "where" the macro is (it can't tell the difference between `<a {{ href }}>` and `<a href="{{ href }}">`, which React can).

Reply:
```suggestion
    <a href="{{ page.url }}">
```